docs/release-schedule.md

```diff
@@ -0,0 +1,9 @@
+Maintenance period
+==================
+
+| Version | Release Date | Maintenance End | End of Life |
+| ------- | ------------ | --------------- | ----------- |
+| v3.x    | 2021-10-14   | 2022-10-01      | 2023-04-01  |
```

Comment:
so are we just going to start this process with v3? As in, in April we release v4 as latest, and v3 becomes previous and there is no version on life-support? I'm definitely ok with that.

From my seed node I see > 200 v2.x.x peers

Reply:
Yes, because we did not have any mechanism to track versions separately it's increasingly harder to support v2 and v1. It is possible of course, we can take last tag v2 and create branch 2.x. We will have to do similar to v3, because a lot of PRs have landed and they need to be sorted.

Now that I think about it, it may be a good experiment and experience to try and support v2 for some time. Just to see the inflexibility and issues with the release process.

Reply:
> Now that I think about it, it may be a good experiment and experience to try and support v2 for some time. Just to see the inflexibility and issues with the release process.

Lets experiment when we have more contributors and maintainers ;-) This is fine as is.